raiden-ts/src/raiden.ts

```diff
@@ -288,6 +289,7 @@ export class Raiden {
     config?: Decodable<PartialRaidenConfig>,
     subkey?: true,
     subkeyOriginUrl?: string,
+    devEnvironment?: string,
```

Comment:
It doesn't feel nice to add this here, as this more devops related than SDK related. 

@andrevmatos Do you see any nicer way? We could parse the contract infos in the CLI and pass them, but then that code would be duplicated.

Reply:
I don´t like it. Passing yet another variable just to choose between deployments seems not a good public API, specially because it´s specific to a network (goerli) and there are already an interface for arbitrary contracts to be fetched: passing UDC address.
CLI or dApp don´t need to parse everything, it can just have the unstable UDC address hardcoded in some `.env` file, env var or `config.json` (maybe conditional to some other envvar).